code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -278,7 +278,7 @@
 		return NUKE_RESULT_WRONG_STATION
 	else if (!disk_rescued && !station_was_nuked && nuke_off_station && syndies_didnt_escape)
 		return NUKE_RESULT_WRONG_STATION_DEAD
-	else if ((disk_rescued || evacuation) && operatives_dead())
+	else if ((disk_rescued && evacuation) && operatives_dead())
```

Comment:
This is inclusive; the extra brackets were only necessary with the or statement.